.github/workflows/docker.yml

```diff
@@ -0,0 +1,39 @@
+name: docker
+
+on:
+  push:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
To make sure we get the latest security patches. Consider the following scenario:

- We build the image for node 14. The current latest version of 14 is 14.18.
- The next day, a security issue is found and node 14.19 gets released
- Schedule trigger fires, rebuilding with the new 14.19 version

Because of the schedule trigger, our images will automatically get the latest version, with the security fix. This is why I'm using broader version numbers like "14" instead of the specific ones we used to have like 10.15.3 (Latest version is [10.24.1](https://nodejs.org/download/release/latest-v10.x/), so we missed a few version there ^^)

Node uses semver, so this /should/ be safe, there wont be any backward incompatible changes.